include/adaflo/sharp_interface.h

```diff
@@ -384,7 +384,13 @@ class LevelSetSolver
   void
   advance_concentration()
   {
+    velocity_solution.update_ghost_values();
+    velocity_solution_old.update_ghost_values();
+    velocity_solution_old_old.update_ghost_values();
     advection_operator->advance_concentration(this->time_stepping.step_size());
+    velocity_solution.zero_out_ghost_values();
```

Comment:
This means that we are not backwards compatible with deal.II version 9.2 where we would have needed `zero_out_ghosts`. It probably won't matter too much because we have not paid attention to it elsewhere and there are no users on deal.II older than master right now (which was different in the past), but we would need to adjust the version here:
https://github.com/kronbichler/adaflo/blob/849a66c4c3afee3fb3a11eca54dec189edf2a807/CMakeLists.txt#L26
In other words, this demonstrates why I am hesitant to frequent interface changes in deal.II - it is a pain to support multiple versions.

Reply:
I have increased the version to 9.3. We are relying (and will be relying thanks to @mschreter) on so many new features from deal.II so that providing backwards compability will be quite hard to accomplish. What do you think?

Reply:
I definitely agree, it is good to make the cut now due to the large new feature set. (I merely used it for demonstration of compatibility.)